tests/integration/test_callback_instrumentation.py

```diff
@@ -0,0 +1,57 @@
+import requests
+
+
+app = """
+ library(dash)
+ library(dashHtmlComponents)
+ library(dashCoreComponents)
+
+ app <- Dash$new()
+
+ app$layout(htmlDiv(
+   list(
+        htmlDiv(children = "Hello world!",
+           id = "messageDiv"
+        ),
+        htmlDiv(id = "emptyDiv")
+    )
+   )
+ )
+
+ app$callback(output = list(id="emptyDiv", property="children"),
+              params = list(input(id="messageDiv", property="children")),
+     function(y) {
+       app$callback_context.record_timing("pancakes", 1.23)
+       Sys.sleep(0.5)
+       return(y)
+     }
+ )
+
+ app$run_server(debug=TRUE)
+"""
+
+def test_rsci001_test_callback_instrumentation(dashr):
```

Comment:
Non-blocking, but in principle this can be `dashr_server` rather than `dashr`, since you don't need the browser, just a server to make a request to.

Reply:
Thanks for the heads-up, fixed in cc44b22.